subcommand/server-acl-init/command_test.go

```diff
@@ -37,18 +37,14 @@ func TestRun_FlagValidation(t *testing.T) {
 	}{
```

Comment:
```
var releaseName = "release-name"
var resourcePrefix = "release-name-consul"
```

can be just resourcePrefix now. Everywhere releaseName is used can be replaced with resourcePrefix